BOTOCORE_README.rst

```diff
@@ -0,0 +1,39 @@
+botocore
+========
+
+
+Background and Implementation
+-------------
+aiobotocore adds async functionality to botocore by replacing certain critical
+methods in botocore classes with async versions.  The best way to see how this
+works is by working backwards from `AioEndpoint._request`.  Because of this tight
+integration aiobotocore is typically version locked to a particular release of
+botocore.
+
+How to upgrade botocore
+-------------
+aiobotocore's file names try to match the botocore files they functionally match.
+For the most part botocore classes are sub-classed with the majority of the
+botocore calls eventually called...however certain methods like
+`PageIterator.next_page` had to be re-implemented so watch for changes in those
```

Comment:
I added this file because the last guy who tried updating support forgot to merge in a few changes